ddcommon/src/profiled_endpoints.rs

```diff
@@ -0,0 +1,72 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use std::ops::AddAssign;
+
+use indexmap::IndexMap;
```

Comment:
There's a few other places already where we use `IndexMap` as well. I'm curious, when is it worth using `IndexMap` over rust's `HashMap`?

Reply:
To be honest, I did not know about `HashMap` and I just followed the usage in the repository

Reply:
Hah, good question! Any insights @morrisonlevi? Rust noobs here ahaha

Reply:
Use `IndexMap` if you need to refer to the inserted thing by number, like in pprof. That's why you see a lot of it it. If you don't need this guarantee, you can use the standard `HashMap`.

Reply:
Use of `HashMap` done